webvtt.go

```diff
@@ -360,11 +385,19 @@ func (s Subtitles) WriteToWebVTT(o io.Writer) (err error) {
 	var c []byte
 	c = append(c, []byte("WEBVTT\n\n")...)
 
+	for _, style := range s.Styles {
```

Comment:
I don't really know whether webvtt allows several `STYLE` tokens, but with that logic, there's a risk of it. Also you need to check `InlineStyle != nil`

I would do something like this:

```go
var style []string
for _, s := range s.Styles {
  if s.InlineStyle != nil && len(s.InlineStyle.WebVTTStyle) > 0 {
    style = append(style,  s.InlineStyle.WebVTTStyle...)
  }
}
if len(style) > 0 {
  c = append(c, []byte(fmt.Sprintf("STYLE\n%s\n\n", strings.Join(style, "\n")))...)
}
```